From: Prarit Bhargava <prarit@redhat.com>
Date: Wed, 4 Jun 2008 09:03:27 -0400
Subject: [ia64] add gate.lds to Documentation/dontdiff
Message-id: 20080604130229.19512.71348.sendpatchset@prarit.bos.redhat.com
O-Subject: [RHEL5 PATCH]: ia64 - Add gate.lds to Documentation/dontdiff
Bugzilla: 449948

Everytime I diff a built tree vs. an unbuilt tree, the gate.lds file appears in
the diff because it isn't in Documentation/dontdiff.

I've submitted this patch upstream 2 or 3 times, and have seen others post the
exact same patch but for some reason it hasn't been accepted upstream.

Successfully tested by me.

Resolves BZ 449948.

diff --git a/Documentation/dontdiff b/Documentation/dontdiff
index 63c2d0c..0bf10c7 100644
--- a/Documentation/dontdiff
+++ b/Documentation/dontdiff
@@ -86,6 +86,7 @@ filelist
 fixdep
 fore200e_mkfirm
 fore200e_pca_fw.c*
+gate.lds
 gen-devlist
 gen-kdb_cmds.c*
 gen_crc32table
